src/helpers/getByAPI.js

```diff
@@ -89,6 +89,16 @@ const getTextInputNodeByDisplayValue = (node, value) => {
   }
 };
 
+const getNodeByTestId = (node, testID) => {
+  try {
+    return typeof testID === 'string'
+      ? testID === node.props.testID
+      : testID.test(node.props.testID);
+  } catch (error) {
+    throw createLibraryNotSupportedError(error);
```

Comment:
We only use `createLibraryNotSupportedError ` when importing something from `react-native`. No need for this try/catch here

Reply:
oh, fair enough. Sorted